pkg/daemon/ceph/client/image_test.go

```diff
@@ -141,6 +141,20 @@ func TestCreateImage(t *testing.T) {
 
 }
 
+func TestResizeImage(t *testing.T) {
+	executor := &exectest.MockExecutor{}
+	context := &clusterd.Context{Executor: executor}
+	executor.MockExecuteCommandWithOutput = func(debug bool, actionName string, command string, args ...string) (string, error) {
+		switch {
+		case command == "rbd" && args[0] == "resize":
+			return "mocked detailed ceph error output stream", fmt.Errorf("some mocked error")
+		}
+		return "", fmt.Errorf("unexpected ceph command '%v'", args)
+	}
+	err := ResizeImage(context, "default", "some-image", "kube", "mon1,mon2,mon3", "/tmp/keyring", 1000000)
+	assert.Nil(t, err)
```

Comment:
For tests where errors are being tested, use `assert.Error` and `assert.NoError`. On that note, there doesn't seem to be a test that tests the error conditions; please add these.

Reply:
`assert.Nil(t, err)` used for checking errors in all `image_test.go` tests, like `TestCreateImage`. 
Should I change it for all assertions in `image_test.go`?

Reply:
I'd like to see all new or modified tests use `assert.Error`/`assert.NoError`. You can change them all if you like, or leave them as is. Please do add a test that will `assert.Error(t, err)` if there is an error in `ExpandImage`. I think this is the only remaining request I have.